### PR TITLE
Tracks Queries

### PR DESCRIPTION
**Pull request by echeipesh**

### CODE REVIEW

File: query-server/server/src/main/scala/com/azavea/ca/server/CAQueryUtils.scala [hunk 1]
```diff
@@ -78,9 +78,9 @@ trait CAQueryUtils {
     val q = new Query(gmFeatureTypeName, query)
     q.getHints.put(org.locationtech.geomesa.accumulo.index.QueryHints.EXACT_COUNT, true)
     q.getHints.put(org.locationtech.geomesa.accumulo.index.QueryHints.LOOSE_BBOX, loose)
-    TestResult.capture(GeoMesaConnection.clusterId, {
-      geomesaFeatureSource().getCount(q)
-    })
+    TestResult.capture(GeoMesaConnection.clusterId) { f =>
+      f(); geomesaFeatureSource().getCount(q)
```
Comment (lossyrob) [inline query-server/server/src/main/scala/com/azavea/ca/server/CAQueryUtils.scala]: this won't measure time to first result correctly, why the change?

Comment (echeipesh) [inline query-server/server/src/main/scala/com/azavea/ca/server/CAQueryUtils.scala]: I see, I misunderstood the way that works.

**Comment by lossyrob** [comment]
Superseded by #36
